config.json

```diff
@@ -792,21 +795,12 @@
     "@typescript-eslint/no-array-constructor": [
       "error"
     ],
-    "no-empty-function": [
```

Comment:
any insights on removing those?

Reply:
I wondered the same :)
I'd prefer to keep it as well. I need to investigate why it has been removed (part of default, or moved to something else? I don't know).
Otherwise, we'll explicily add it, but I did not want to resort to this yet.

Reply:
So actually, this rule moved to the stylistic recommended rule. I updated the PR and added stylistic :)